src/App.js

```diff
@@ -10,6 +10,7 @@ import { ExtraInfoPage } from './page/ExtraInfoPage';
 import { HomePage } from './page/HomePage';
 import { MentorPage } from './page/MentorPage';
 import { NextEditionPage } from './page/NextEditionPage';
+import { WhatWeDoPage } from './page/WhatWeDo';
```

Comment:
Zou je de filename willen veranderen naar `WhatWeDoPage`, dan is de benaming inlijn met de rest...

Reply:
Tuurlijk, done, helemaal eens dat dat veel netter is, geen idee waarom ik dat gemist heb...

Valt een beetje buiten deze issue, maar heb je suggesties voor goede linters en of autoformatters?

Reply:
Dat heb ik zeker! Ik ben daar namelijk een groot fan van.
De linter is `eslint` ik zal een issue aanmaken voor het toevoegen van een linter.
Daarnaast is er ook een autoformatter, genaamd `prettier`. Ik zal ook hiervoor een issue aanmaken zodat deze kan worden toegevoegd.